.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+# We do not use Github for issues!
+
+Please do not create issues here.
```

Comment:
I think that would make sense, and maybe mention that only OTW volunteers should do it, and everyone else should report bugs to Support?